frontend/src/plugins/veeValidate/lessThanOrEqual_date.js

```diff
@@ -0,0 +1,15 @@
+export default (dayjs, i18n) => ({
+  params: ['max'],
+  /**
+   * @param   {string}  value Dater value in local string format
+   * @param   {string}  max   comparison valye in local string format
+   * @returns {bool}          validation result
+   */
+  validate: (value, { max }) => {
```

Comment:
A test would be nice, we already have them for `greaterThan` and `greaterThanOrEqual`